Source/stores.cpp

```diff
@@ -2169,6 +2169,8 @@ void BarmaidEnter()
 	case 14:
 		stextflag = STORE_NONE;
 		IsStashOpen = true;
+		// Forces the page to be refreshed (are items valid to use ...)
+		Stash.SetPage(Stash.GetPage());

```

Comment:
Having a `Stash::RefreshItemStatFlags()` (or similar) method would be good here instead of a weird looking call to set the page to the current page :D

Reply:
Thanks. I added the method. 🙂